app/src/main/java/vn/mbm/phimp/me/PhotoSelect.java

```diff
@@ -309,7 +309,7 @@ public void run() {
 	 		        
 	 			}
 	    	}catch(RuntimeException runex){
-	    		this.onCancelled();
+	    		//this.onCancelled();
```

Comment:
Should this be uncommented, I guess it is a good UI to stop the process when an exception has occurred.